code/game/machinery/doors/airlock.dm

```diff
@@ -500,6 +501,81 @@ About the new airlock wires panel:
 				sleep(6)
 				update_icon(AIRLOCK_CLOSED)
 
+
+/obj/machinery/door/airlock/proc/change_paintjob(obj/item/airlock_painter/W, mob/user)
+	if(!paintable)
+		to_chat(user, "<span class='warning'>You cannot paint this type of airlock.</span>")
+		return FALSE
+
+	var/list/optionlist
+	if(airlock_material == "glass")
+		optionlist = list("Standard", "Public", "Engineering", "Atmospherics", "Security", "Command", "Medical", "Research", "Science", "Mining", "Maintenance", "External", "External Maintenance")
+	else
+		optionlist = list("Standard", "Public", "Engineering", "Atmospherics", "Security", "Command", "Medical", "Research", "Freezer", "Science", "Mining", "Maintenance", "External", "External Maintenance")
```

Comment:
The only extra option in non-glass is the freezer. What do you think about
```js
var/list/optionlist = list( < everything except freezer > )
if(airlock_material != "glass")
  optionlist |= "Freezer"
```
?

If freezer is not the only one missing - that's the more reason to make the distinction between the two lists clearer.

Reply:
yeah that is a better option, I'll change it